src/navigation/types.ts

```diff
@@ -0,0 +1,9 @@
+export enum rootScreens {
+  BottomTabs = "BottomTabs",
+  EmptyPage = "EmptyPage",
+  Home = "Home",
+}
+
+export type RootStackParamList = {
+  [rootScreens.BottomTabs]: undefined;
+};
```

Comment:
Update `rootScreens ` enum to include the new screen, and then use that enum in the `RootStackParamList` and in the navigator.